sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatRequestUserMessage.java

```diff
@@ -148,7 +146,7 @@ public static ChatRequestUserMessage fromJson(JsonReader jsonReader) throws IOEx
                 String fieldName = reader.getFieldName();
                 reader.nextToken();
                 if ("content".equals(fieldName)) {
-                    content = reader.getNullable(nonNullReader -> BinaryData.fromObject(nonNullReader.readUntyped()));
+                    content = BinaryData.fromBytes(reader.getBinary());
```

Comment:
If `content` can be `null` I think this will still need `reader.getNullable`. But I think this needs more logic here if the content can be null, string, array, or object.

```java
if (reader.currentToken() == JsonToken.STRING) {
    content = BinaryData.fromString(reader.getString());
} else if (reader.currentToken() == JsonToken.START_OBJECT) {
    content = BinaryData.fromObject(reader.readMap(JsonReader::readUntyped));
} else if (reader.currentToken() == JsonToken.START_ARRAY)) {
    content = BinaryData.fromObject(reader.readArray(JsonReader::readUntyped));
} else if (reader.currentToken() == JsonToken.NULL) {
    content = null;
} else {
    // invalid state? Probably throw?
}
```